exec/src/coqstr.ml

```diff
@@ -44,32 +44,34 @@ let char_to_coqascii char =
   in
   mkApp (cAscii, of_list @@ List.rev (int_to_booll 8 src []))
 
-(* val bytes_fold_chars: bytes -> ('b -> char -> 'b) -> 'b -> 'b *)
-let bytes_fold_chars_rev str f =
+(* val fold_chars: 'a -> int -> ('a -> int -> char) -> ('b -> char -> 'b) -> 'b -> 'b *)
```

Comment:
I am not sure that this is a good idea to write types in comments since they are not checked by the compiler what about using:
```suggestion
let fold_chars
: 'a -> int -> ('a -> int -> char) -> ('b -> char -> 'b) -> 'b -> 'b =
fun buffer size get f = ...
```
instead?